doc/sphinx/proofs/writing-proofs/reasoning-inductives.rst

```diff
@@ -237,44 +237,54 @@ analysis on inductive or co-inductive objects (see :ref:`variants`).
 Induction
 ---------
 
-.. tacn:: induction @term
-   :name: induction
+.. tacn:: induction {+, @induction_clause } {? @induction_principle }
```

Comment:
I think it would make sense to move the definition of `induction_principle` here.